testdata/scripts/lines.txt

```diff
@@ -0,0 +1,62 @@
+garble -debugdir=.obf-src build
+
+env TINY_PATTERN='^\/\/line :1$'
+env DEFAULT_PATTERN='^\/\/line \w\.go:[1-9][0-9]*$'
+env DEFAULT_STACK_PATTERN='^\t\w\.go:[1-9][0-9]*(\s\+0x[0-9a-f]+)?'
+env TINY_STACK_PATTERN='^\t\?\?:[0-9][0-9]*(\s\+0x[0-9a-f]+)?$'
+
+# Default mode

```

Comment:
this should be in the second half of the script, and be skipped if `-short` is used. otherwise this will slow down `go test -short` twice as much